### PR TITLE
feat(landing): support geojson reprojection into NZTM with maplibre

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: packages/landing/src/components/debug.tsx [hunk 1]
```diff
@@ -109,52 +114,108 @@ export class Debug extends Component<{ map: maplibre.Map }, { lastFeatureId: str
...
 
+  trackMouseMove(layerId: string): void {
+    const sourceId = `${layerId}_source`;
+    const layerFillId = `${sourceId}_fill`;
+    const map = this.props.map;
+
+    // const source = '';
+    let lastFeatureId: string | number | undefined;
+    map.on('mousemove', layerFillId, (e) => {
+      const features = e.features;
```
Comment (Wentao-Kuang) [inline packages/landing/src/components/debug.tsx:125]: Are we got an Id for each feature?

Comment (blacha) [reply packages/landing/src/components/debug.tsx:125]: I force added a featureId further down `f.id = id++`

Comment (Wentao-Kuang) [reply packages/landing/src/components/debug.tsx:125]: Oh, I just saw it.

File: packages/landing/src/tile.matrix.ts [hunk 2]
```diff
@@ -56,7 +56,40 @@ export function locationTransform(
...
   const tile = { x: center.x / tileMatrix.tileSize, y: center.y / tileMatrix.tileSize, z: Math.round(location.zoom) };
-  const mapboxTile = targetTileMatrix.tileToSource(tile);
-  const [lon, lat] = Projection.get(targetTileMatrix).toWgs84([mapboxTile.x, mapboxTile.y]);
+  const wmTile = targetTileMatrix.tileToSource(tile);
+  const [lon, lat] = Projection.get(targetTileMatrix).toWgs84([wmTile.x, wmTile.y]);
   return { lon: Math.round(lon * 1e8) / 1e8, lat: Math.round(lat * 1e8) / 1e8, zoom: location.zoom };
 }
+
+/** Re project WGS84 coordinates into the target tile matrix for use within maplibre */
+export function projectToGoogle(location: [number, number], targetTileMatrix: TileMatrixSet): [number, number] {
```
Comment (Wentao-Kuang) [inline packages/landing/src/tile.matrix.ts:65]: I think we have existing function doing the same thing. `locationTransform()` in `tile.matrix.ts`

Comment (blacha) [reply packages/landing/src/tile.matrix.ts:65]: Hmm when I first tried it it was giving me very weird coordiantes, but looking at it I think If i do `NZTM200QuadTms, GoogleTms` it should maybe work let me try it

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**